src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -277,6 +278,13 @@ const formConfig = {
           uiSchema: unemployabilityFormIntro.uiSchema,
           schema: unemployabilityFormIntro.schema,
         },
+        uploadForm4192: {
+          title: 'Upload Form 4192',
+          path: 'new-disabilities/4192-upload',
+          depends: formData => formData['view:upload4192Choice'] === 'upload',
```

Comment:
For consistency, can this be a function in the utils file that sits next to `isUploadingPtsdForm` and the others?

Reply:
I think our thought process here was that we were only going to be creating util functions when something was going to be either reused by multiple page configs or had more than one condition.

Would you rather us just create util functions for the depends handlers as a general rule? I'm totally down with doing it either way.

Reply:
I think if there's a specific value that we're checking against (e.g., `upload` here), it would make sense to throw it into `utils`. Doesn't make a huge difference either way, but the fewer strings we have floating around in the config, the better, probably.